src/vse_sync_pp/analyzers/ppsdpll.py

```diff
@@ -4,27 +4,42 @@
 
 from .analyzer import Analyzer
 
-class PhaseOffsetTimeErrorAnalyzer(Analyzer):
+class TimeErrorAnalyzer(Analyzer):
```

Comment:
Given that the two `TimeErrorAnalyzer` test implementations look the same, we should make a base class. The material differences look like `id_`, `parser` and which column to select for testing/reporting.